tests/test_meshtastic_utils.py

```
@@ -5318,10 +5389,10 @@ def submit_side_effect(_func, *_args, **_kwargs):
             self.assertEqual(len(warning_calls), MAX_TIMEOUT_RETRIES_INFINITE)
 
     @patch("mmrelay.meshtastic_utils.time.sleep")
-    def test_connect_meshtastic_skips_scan_recovery_for_auto_reconnect_interfaces(
+    def test_connect_meshtastic_does_not_scan_after_ble_errors_auto_reconnect(
         self, _mock_sleep
```

Ruff (0.15.9)

[warning] Fixture `_mock_sleep` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)